src/apis/user/user.resolver.ts

```diff
@@ -17,13 +17,26 @@ export class UserResolver {
 
   //닉네임 체크
   @Mutation(() => String)
-  async checkNickName(
+  async checkNickname(
     @Args('nickname') nickname: string, //
   ) {
-    const findNickName = await this.userService.findOne({ nickname });
-    if (findNickName) {
+    const findNickname = await this.userService.findOne({ nickname });
+    if (findNickname) {
       return false;
-    } else return true;
+    }
+    return true;
+  }
+
+  //이메일 체크
+  @Mutation(() => String)
+  async checkEmail(
+    @Args('email') email: string, //
+  ) {
+    const findEmail = await this.userService.findOne({ email });
+    if (findEmail) {
+      return false;
+    }
+    return true;
```

Comment:
```suggestion
    return !!findEmail;
```
해당 구문은 이렇게 축약하는것과 동일한 동작을 하는데 취향이 많이 갈립니다 ㅎㅎ

Reply:
오 이렇게 간단히 해도 같은 동작이라니 배워갑니다 감사합니다..

Reply:
아니면 return findEail ? false : true; 이렇게 삼항연산자는 어떨까요? 둘중 어떤것을 더 선호하세요?!